RudderFirebase.podspec

```diff
@@ -20,5 +20,5 @@ Pod::Spec.new do |s|
     s.swift_version = '5.3'
 
```

Comment:
Please correct the file name to `LICENSE.md` over [here](https://github.com/rudderlabs/rudder-integration-firebase-swift/blob/cd308e9a04f91e4fd9180a20b7909e40173d67f4/RudderFirebase.podspec#L9).

Reply:
updated file name